server/routerlicious/packages/services/src/mongoExceptionRetryRules/mongoError/index.ts

```diff
@@ -117,7 +128,7 @@ class RequestTimedOutWithRateLimit implements IMongoExceptionRetryRule {
 class ServiceUnavailableRule implements IMongoExceptionRetryRule {
     private static readonly errorDetails = "Response status code does not indicate success: ServiceUnavailable (503)";
     match(error: any): boolean {
-        return error.code === 50
+        return error.code === 1
```

Comment:
Was error code 50 not appropriate here?

Reply:
That is right, I copy pasted wrong stuff. The sample exception for service unavailable is :
> {"driver":true,"name":"MongoError","index":0,"code":1,"errmsg":"[ActivityId=d....] Error=1, Details='Response status code does not indicate success: ServiceUnavailable (503); Substatus: 20001; ActivityId: ....